src/construction/casting.rs

```diff
@@ -198,8 +191,8 @@ impl ApInt {
         if target_width < actual_width {
             return Error::extension_bitwidth_too_small(target_width, actual_width)
                 .with_annotation(format!(
-                    "Cannot sign-extend bit-width of {:?} to {:?} bits. \
-                     Do you mean to truncate the instance instead?",
+                    "Cannot sign-extend bit-width of {:?} to {:?} bits. Do you mean to truncate \
```

Comment:
Are these reformattings of comments the result of rustfmt auto formatting?

Reply:
Yes. Do you not like them?